tools/ktx/command_create.cpp

```diff
@@ -1351,10 +1351,12 @@ void CommandCreate::executeCreate() {
             }
 
             if (options.normalize) {
-                if (target.format().transfer() != KHR_DF_TRANSFER_LINEAR) {
+                if (target.format().transfer() != KHR_DF_TRANSFER_UNSPECIFIED && target.format().transfer() != KHR_DF_TRANSFER_LINEAR) {
                     fatal(rc::INVALID_FILE,
-                        "Input file \"{}\" transfer function is not linear. Normalize is only available for linear images. "
-                        "Use --assign-oetf=linear or --convert-oetf=linear to convert to linear if required.",
+                        "Input file \"{}\" The transfer function to be applied to the created texture is neither linear nor none. Normalize is only available for linear images. "
```

Comment:
I find it a little confusing saying just "linear" in the second sentence. Suggest changing to "... only available for these transfer functions."

Reply:
I have rewored this area with more detailed error message https://github.com/KhronosGroup/KTX-Software/pull/977/commits/ae46489470217b430e1f957aa3c4a5401d90fa69